esp/src/src-react/components/DaliCount.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from "react";
+import { DefaultButton, DetailsList, DetailsListLayoutMode, IColumn } from "@fluentui/react";
+import { SizeMe } from "react-sizeme";
+import { csvParse } from "d3-dsv";
+import { DaliService } from "@hpcc-js/comms";
+import { scopedLogger } from "@hpcc-js/util";
+import { TableGroup } from "./forms/Groups";
+import nlsHPCC from "src/nlsHPCC";
+import { HolyGrail } from "../layouts/HolyGrail";
+
+const logger = scopedLogger("src-react/components/DaliCount.tsx");
+
+const myDaliService = new DaliService({ baseUrl: "" });
+
+interface CountRequestProps {
+
+}
+
+export const DaliCount: React.FunctionComponent<CountRequestProps> = ({
+
+}) => {
+
+    const [columns, setColumns] = React.useState<IColumn[]>([]);
+    const [items, setItems] = React.useState<any[]>([]);
+    const [path, setPath] = React.useState<string>("");
+
+    const onSubmit = React.useCallback(() => {
+        myDaliService.Count({ Path: path }).then(response => {
+            const data = csvParse(response.Result);
```

Comment:
The problem here is that response.Result is a number, rather than a string. I don't remember which, but I think you had a component in your earlier set for DaliAdmin where the response didn't need to use csvParse either, because it was just a single value, a boolean or something.